src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -63,3 +63,17 @@ <h5>{% trans "Prefill plugins" %}</h5>
         {% endfor %}
     </ul>
 {% endif %}
+
+{% if broken_configurations %}
+    <h5>{% trans "Broken configuration" %}</h5>
+    <ul>
+        {% for config in broken_configurations %}
+            <li>
+                {% blocktranslate with name=config.config_name message=config.exception_message trimmed %}
+                    Configuration for '{{ name }}' is broken (<i>{{ message }}</i>)<br/>
```

Comment:
In the `BRKValidatorCheck` class, I see some error messages with and without a dot. So we probably need to find a way to add one if required